crates/mfem-sys/build.rs

```diff
@@ -77,45 +78,46 @@ impl MfemConfig {
         let dst = dst.expect("Bundled MFEM not found.");
 
         #[cfg(not(feature = "bundled"))]
-        let dst = dst.expect("Pre-installed MFEM not found. You can use `bundled` feature if you do not want to install MFEM system-wide.");
+        let dst = dst.expect("Pre-installed MFEM not found.  You can use MFEM_DIR to point to its location. Alternatively, you can use `bundled` feature if you do not want to install MFEM system-wide.");
 
         let cfg = std::fs::read_to_string(dst.join("share").join("mfem_info.txt"))
             .expect("Something went wrong when detecting MFEM.");
 
         let mut version: Option<semver::Version> = None;
-        let mut mfem_libs: Vec<String> = vec![];
-        let mut include_dirs: Vec<std::path::PathBuf> = vec![];
-        let mut library_dir: Option<std::path::PathBuf> = None;
-        let mut cxx_flags: Vec<String> = vec![];
+        let mut c = MfemConfig {
```

Comment:
nit: Please use a more descriptive name, such as `mfem_config`.
```suggestion
        let mut mfem_config = MfemConfig {
```